symphony-bdk-core/src/main/resources/template/default_help_command.ftl

```diff
@@ -0,0 +1,7 @@
+<messageML>
```

Comment:
aren't we going to have the issue that @symphony-thibault mentioned? (handlebar being used instead)

Reply:
I found the `symphony-bdk-template:symphony-bdk-template-freemarker` in the dependencies of the core module, so I guess freemarker used by default in the core

Reply:
with a limited scope testRuntimeOnly

Reply:
Developer can exclude this dependency to use `symphony-bdk-template:symphony-bdk-template-handlebars`

Reply:
so I guess we should consider not to use template in this case? maybe building a string in order is enough?